scripts/customization-surface.sh

```
@@ -0,0 +1,217 @@
+cs_main() {
+  local cmd="${1:-}"; shift || true
+  case "$cmd" in
+    resolve)
+      local p="${1:?usage: resolve <path>}"
+      cs_match_g "$p"
+      printf '%s\t%s%s\n' "$p" "$CS_M_OWNER" "${CS_M_STRAT:+  (strategy: $CS_M_STRAT)}"
+      ;;
+    report)
+      local f; declare -A cnt=()
+      while IFS= read -r f; do
+        cs_match_g "$f"; cnt[$CS_M_OWNER]=$(( ${cnt[$CS_M_OWNER]:-0} + 1 ))
```

Shellcheck (0.11.0)

[info] Modification of cnt is local (to subshell caused by pipeline).

(SC2030)